src/pkcs15init/pkcs15-lib.c

```diff
@@ -927,7 +927,10 @@ sc_pkcs15init_add_app(struct sc_card *card, struct sc_profile *profile,
 			free(p15card->tokeninfo->label);
```

Comment:
nit: once we touch this, I would replace the above two lines with just the free (removing the NULL check `if (p15card->tokeninfo->label)`):
```suggestion
		free(p15card->tokeninfo->label);
```

Reply:
well, did that first and decided that it is too far off. anyway, I'll add that in :wink: